provider/util.go

```diff
@@ -74,17 +75,26 @@ func normalizeMonitor(tpl map[string]interface{}) {
 }
 
 func normalizeMonitorTriggers(triggers []interface{}) {
+	trigger_types := []string{"alerting_trigger", "anomaly_detector_trigger", "bucket_level_trigger", "document_level_trigger", "query_level_trigger"}
 	for _, t := range triggers {
 		if trigger, ok := t.(map[string]interface{}); ok {
 			delete(trigger, "id")
-
-			if actions, ok := trigger["actions"].([]interface{}); ok {
-				normalizeMonitorTriggerActions(actions)
```

Comment:
Why are you removing this part of the normalization?

Reply:
I didn't see it present in any of the alerts that we had in our system, so it seemed spurious to keep it there. Do you want me to add it back?

Reply:
It was introduced here: https://github.com/phillbaker/terraform-provider-elasticsearch/pull/164 can you see if that's still an issue?

Reply:
Checked it out and it's not an issue with the newer opensearch monitors. 

The opensearch alerting plugin changes aren't using the same syntax/representation as the elasticsearch, so we can delete this without worries.